package.json

```diff
@@ -10,7 +10,7 @@
     "@fontsource/ibm-plex-mono": "^4.5.13",
     "@fontsource/open-sans": "^4.5.14",
     "@fontsource/work-sans": "^4.5.14",
-    "gatsby": "^5.9.0",
+    "gatsby": "^5.10.0",
     "gatsby-plugin-catch-links": "^5.4.0",
     "gatsby-plugin-csp": "^1.1.4",
```

Comment:
From the PR title I was expecting to see something about engine.io also. Was it missed ? or is this just about some gatsby bump, and the title is wrong ?